api/src/main/java/io/opencensus/metrics/MetricRegistry.java

```diff
@@ -78,33 +65,15 @@ static MetricRegistry newNoopMetricRegistry() {
   private static final class NoopMetricRegistry extends MetricRegistry {
 
     @Override
-    public <T> void addLongGauge(
-        String name,
-        String description,
-        String unit,
-        LinkedHashMap<LabelKey, LabelValue> labels,
-        T obj,
-        ToLongFunction<T> function) {
-      Utils.checkNotNull(name, "name");
-      Utils.checkNotNull(description, "description");
-      Utils.checkNotNull(unit, "unit");
-      Utils.checkNotNull(labels, "labels");
-      Utils.checkNotNull(function, "function");
+    public NoopLongGaugeMetric addLongGaugeMetric(
+        String name, String description, String unit, List<LabelKey> labelKeys) {
+      return new NoopLongGaugeMetric(name, description, unit, labelKeys);
```

Comment:
Is it possible to create a static instance of `NoopLongGaugeMetric` and always return the same instance?